src/features/content-explorer/content-explorer/ContentExplorer.js

```diff
@@ -284,11 +312,11 @@ class ContentExplorer extends Component {
     };
 
     handleItemClick = ({ event, index }) => {
-        const { contentExplorerMode, items, onSelectItem } = this.props;
+        const { contentExplorerMode, onSelectedItems, items, onSelectItem } = this.props;
         const { selectedItems } = this.state;
         const item = items[index];
 
-        if (item.isDisabled || item.isLoading) {
+        if (item.isDisabled || item.isLoading || item.isActionDisabled) {
```

Comment:
wouldn't this change existing behavior?

i.e. if the parent folder has `isActionDisabled` but the subfolders do not, then we should allow the user to click into that folder to select the subfolders

there's a demo of that here: https://opensource.box.com/box-ui-elements/#/Features?id=content-explorer
with the `Can't choose but can drill in` item

<img width="632" alt="Screen Shot 2023-02-15 at 11 37 11 PM" src="https://user-images.githubusercontent.com/7311041/219298713-e2a58b95-949e-4183-a828-c52c5a57c685.png">
<img width="635" alt="Screen Shot 2023-02-15 at 11 37 17 PM" src="https://user-images.githubusercontent.com/7311041/219298733-6d31cb54-74c7-486d-a7be-01f1f7e9573d.png">

Reply:
although now I think I might misunderstand how `handleItemClick` is being called

Reply:
yup I think i understood this incorrectly. what you have should be right.

Reply:
I might be a little confused. I thought this would still be fine since this is the handleitemclick and not handleItemNameClick. So users can still click into this folder to see subcontents even if they are action disabled. I thought then a new items array gets passed in as the items prop for ContentExplorer, where then those subcontents will have their own values of action disabled or disabled and they can be selected or not

Reply:
Oops sorry replied to this a tad too late. Glad we are one same page though